landoapi/hg.py

```diff
@@ -391,14 +391,21 @@ def format(self) -> Optional[List[str]]:
         # Update the working directory to the latest change.
         self.run_hg(["update", "-C", "-r", "tip"])
 
-        formatted_replacements = check_fix_output_for_replacements(fix_output)
-        logger.info(
-            f"revisions were reformatted: {', '.join(formatted_replacements)}"
-            if formatted_replacements
-            else "autoformatting did not change any revisions"
+        # Exit if no revisions were reformatted.
+        pre_formatting_hashes = check_fix_output_for_replacements(fix_output)
+        if not pre_formatting_hashes:
+            logger.info("autoformatting did not change any revisions")
```

Comment:
Unsure if this log line is valuable, but that's up to you.

Reply:
I agree it's not super useful - dumping it.